eng/pipelines/coreclr/templates/crossdac-build.yml

```diff
@@ -13,31 +13,48 @@ steps:
     - script: $(Build.SourcesDirectory)/src/coreclr/build-runtime$(scriptExt) $(buildConfig) $(archType) -hostarch ${{ parameters.hostArchType }} -ci -os alpine -cmakeargs "-DCLR_CROSS_COMPONENTS_BUILD=1" -ninja $(officialBuildIdArg) -component crosscomponents
       displayName: Build Cross OS Linux-musl DAC for Windows
 
-    - task: CopyFiles@2
-      displayName: Gather CrossDac Artifacts (Linux)
-      inputs:
-        SourceFolder: $(buildLinuxDacRootFolderPath)
-        Contents: |
-          **/*
-          !**/sharedFramework/**/*
-        TargetFolder: $(buildLinuxDacStagingPath)
+    - powershell: |
+        function CopyAndVerifyCrossOsAssets {
+          [CmdletBinding()]
+          param (
+            [Parameter(Mandatory)][string]$crossDacDir,
+            [Parameter(Mandatory)][string]$targetDir
+          )
 
-    - task: CopyFiles@2
-      displayName: Gather CrossDac Artifacts (Linux_musl)
-      inputs:
-        SourceFolder: $(buildMuslDacRootFolderPath)
-        Contents: |
-          **/*
-          !**/sharedFramework/**/*
-        TargetFolder: '$(buildMuslDacStagingPath)'
+          if ('$(archType)' -ne '${{ parameters.hostArchType }}')
```

Comment:
Should/can we use `${{ parameters.archType }}` instead of `$(archType)`?

Reply:
Somehow I missed that, thanks @am11